cmd/context/context.go

```diff
@@ -222,7 +218,66 @@ func Init(ctx context.Context) error {
 			return err
 		}
 	}
-	os.Setenv("OKTETO_USERNAME", okteto.Context().Username)
-	os.Setenv("OKTETO_NAMESPACE", okteto.Context().Namespace)
+	if okteto.IsOktetoContext() {
+
+		secretsAndKubeCredentials, err := getSecretsAndCredentials(ctx)
+		if err != nil {
+			return err
+		}
+		if err := updateContext(secretsAndKubeCredentials.Credentials); err != nil {
+			log.Info(err)
+		}
+		setSecrets(secretsAndKubeCredentials.Secrets)
+
+		os.Setenv("OKTETO_USERNAME", okteto.Context().Username)
+		os.Setenv("OKTETO_NAMESPACE", okteto.Context().Namespace)
```

Comment:
I would keep `OKTETO_NAMESPACE` outside of the if block, it is valid for non okteto clusters